templates/concert-ticket/templates/main.tpl

```diff
@@ -0,0 +1,173 @@
+                <text-chunk font="helvetica" font-size="9"> it is the top venue for people of all ages to come together for one common purpose: </text-chunk>
+                <text-chunk font="helvetica" font-size="9">to congregate with other rock fans from across the world, in a place where rock music still matters.</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Featuring the very best of active and classic rock and legendary names in Rock Music, entertainment and experience are the first priority.</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Aerosmith, Iron Maiden, Avenged Sevenfold, Kiss, Motley Crue, Fleetwood Mac, Tom Petty, Kid Rock, Shinedown,</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Five Finger Death Punch, Rob Zombie, Korn &amp; many more rock legends have graced this permanent Main Stage over the course of the last two and a half decades.</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+    <table-cell vertical-align="top">
+        <image src="path('./templates/res/map.png')" height="160" width="100" margin="5 0 0 0"></image>
+    </table-cell>
+    <table-cell vertical-align="top">
+        <division>
+            <paragraph text-align="top" margin="0 0 0 0">
+                    <text-chunk font="helvetica-bold" font-size="12">How to find us</text-chunk>
+            </paragraph>
+            <paragraph text-align="top" margin="5 0 0 0" line-height="1.3">
+                <text-chunk font="helvetica" font-size="9">Concert Hall is located three blocks west of the park and three blocks north the library.</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Free parking is available nearby, which fills up close to showtime, and on the streets around.</text-chunk>
+                <text-chunk font="helvetica" font-size="9">Bike racks are located outside the main entrance tothe Hall.</text-chunk>
+            </paragraph>
+            <paragraph text-align="top" margin="10 0 0 0">
+                <text-chunk font="helvetica-bold" font-size="12">Learn More At</text-chunk>
+            </paragraph>
+            <paragraph margin="5 0 0 0">
+                <text-chunk font="helvetica" font-size="9">Facebook /ConcertHall</text-chunk>
+            </paragraph>
+            <paragraph margin="2 0 0 0">
+                <text-chunk font="helvetica" font-size="9">Twitter @ConcertHall</text-chunk>
+            </paragraph>
+            <paragraph margin="2 0 0 0">
+                <text-chunk font="helvetica" font-size="9">Instagram @ConcertHall</text-chunk>
+            </paragraph>
+        </division>
+    </table-cell>
+</table>
+
+<table columns="2">
+    <table-cell vertical-align="middle">
+        <division margin="0 0 0 10">
+            <paragraph text-align="top" margin="0 0 0 0">
```

Comment:
`top` is not a valid value for the `text-align` attribute of paragraphs.
Also, the default value for the `margin` attribute is `0`. No need to specify it in this case.